CMakeLists.txt

```diff
@@ -67,7 +67,11 @@ ELSE()
   ADD_COMPILE_FLAG("-Wextra")
   ADD_COMPILE_FLAG("-Wno-unused-parameter")
   ADD_COMPILE_FLAG("-fno-omit-frame-pointer")
-  ADD_COMPILE_FLAG("-fPIC")
+  IF(WIN32)
+    ADD_COMPILE_FLAG("-D_GNU_SOURCE")
```

Comment:
Without it, MinGW job fails at 76% like https://ci.appveyor.com/project/jasonwilliams200OK/binaryen/build/1.0.36/job/36pvx89h7g52g09n#L110.